docs/example_use.rst

```diff
@@ -0,0 +1,139 @@
+.. Copyright 2017 Covata Limited or its affiliates
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+Example Usage
+=============
+
+These examples assume a folder called ``~/keystore`` is present with
+``passPhrase`` as the password. Each example code-snippet is self-contained
+and runnable.
+
+Initialisation
+--------------
+- Initialising SDK Client (Configuration via Parameters)
+
+.. code:: python
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+
+Identity
+--------
+
+- Create an identity
+.. code:: python
+
+   from covata import delta
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   client.create_identity()
+
+- Get your own identity
+.. code:: python
+
+   from covata import delta
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   identity = client.get_identity("8e91cb8c-1ea5-4b69-bedf-9a14940cce44")
+
+- Get a different identity
+.. code:: python
+
+   from covata import delta
```

Comment:
I feel the way it is done right now makes our doco prone to mistakes as there's no direct link to the code. why not use literalinclude instead? this way we can put each example in one .py file which has IDE support for linting, syntax highlighting and code completion.

```
- Getting a different identity

  .. literalinclude:: examples/get_different_identity.py
```